src/CoreWCF.Http/src/CoreWCF/Channels/HttpTransportBindingElement.cs

```diff
@@ -257,5 +259,29 @@ internal static AuthenticationSchemes GetEffectiveAuthenticationSchemes(Authenti
                 return currentAuthenticationSchemes & hostSupportedAuthenticationSchemes;
             //}
         }
+        public ExtendedProtectionPolicy ExtendedProtectionPolicy
+        {
+            get
+            {
+                return _extendedProtectionPolicy;
+            }
+            set
+            {
+                if (value == null)
+                {
+                    throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(value));
+                }
+
+                if (value.PolicyEnforcement == PolicyEnforcement.Always &&
+                    !System.Security.Authentication.ExtendedProtection.ExtendedProtectionPolicy.OSSupportsExtendedProtection)
+                {
+                    // ExceptionHelper.PlatformNotSupported(SR.ExtendedProtectionNotSupported);
+                }
```

Comment:
Why is this line uncommented? If the platform doesn't support it so we should throw if turning this on.